### DISCUSSION TITLE
Add multiple figure properties (fig-alt, fig-align)

### DISCUSSION THREAD

**Discussion by e-alizadeh** [quarto-dev/quarto-web#419] [Q&A]
Two separate issues here: 
1) how can I add multiple figure properties. I've tried the following
`![abc](img/image.png){fig-alt="hello", fig-align="right"}` and `![abc](img/image.png){fig-alt="hello"}{fig-align="right"}` but neither worked. Instead, I'm getting the following  
![align_alt](https://user-images.githubusercontent.com/30320825/192682290-3cfeaa37-9da6-4062-93f2-ad2b3c7d3bbe.png)

2) if I set up `fig-align: center` in `_quarto.yml`, shouldn't it possible to override it by explicitly passing `fig-align` attribute? If yes, then I probably should open a bug report here?

Thanks in advance

**Comment by dragonstyle** [comment]
I think the issue is the comma - when I write the following locally, as you'd expect I get the right aligned figure with the correct alt text and caption:

```markdown
![The Boxer is a medium to large, short-haired dog breed.](dog.png){fig-align="right" fig-alt="This is the alt text"}
```

Overriding appears to be working for me as well when I test it locally.

**Reply by e-alizadeh** [reply]
Thanks @dragonstyle. The issue was indeed the comma. The overriding works for me after that too.
